consensus/core/src/commit_observer.rs

```diff
@@ -234,6 +236,8 @@ impl CommitObserver {
                     .node_metrics
                     .commit_observer_last_recovered_commit_index
                     .set(last_sent_commit_index as i64);
+
+                tokio::task::yield_now().await;
```

Comment:
Maybe we don't need to do it on very `sub dag send` , but having like this for now